src/core/SIP/Channels/SIPTLSChannel.cs

```diff
@@ -121,7 +121,13 @@ protected async Task OnAcceptAsync(SIPStreamConnection streamConnection)
                     cts.Cancel();
 
                     logger.LogDebug("SIP TLS Channel successfully upgraded accepted client to SSL stream for {ListeningSIPEndPoint}<-{RemoteSIPEndPoint}.", ListeningSIPEndPoint, streamConnection.RemoteSIPEndPoint);
-
+                    

```

Comment:
Wouldn't this block be be better directly after cts.Cancel()? It'd be confusing to see a connection success log meesage followed by a connection error message.